tuareg-opam.el

```diff
@@ -313,6 +313,9 @@ characters \\([0-9]+\\)-\\([0-9]+\\): +\\([^\n]*\\)$"
   (setq indent-tabs-mode nil)
   (setq-local require-final-newline mode-require-final-newline)
   (smie-setup tuareg-opam-smie-grammar #'tuareg-opam-smie-rules)
+  ;; Explicit variable declarations to avoid Emacs 24 warnings
```

Comment:
We could, but wouldn't
```
(when (< emacs-major-version 25)
  (defvar this)
  (defvar that))
```
just raise more questions: what _is_ really the scoping of `defvar`? It looks like the `defvar`s have to be 'executed' in order to have effect, which isn't true, so the code would really be a lie.

(This is an open problem, actually – I think the scoping constructs for `defvar` are `let`, `let*` and `lambda`, which means that sometimes we have to add dummy variables just for this purpose which clashes with optimisations that remove unused variables...)

We would likely grep for "24" or "25" when removing support for Emacs 24, so this would be found.
I'll add blank lines around the `defvar`s here so that they are more easily seen as a group.